uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Form/AbsencePeriod.php

```diff
@@ -137,6 +144,11 @@ private function addFields() {
       $this->getDAOFieldAttributes('weight'),
       true
     );
+
+    //This element will have its value set if the user
```

Comment:
```js
// This element is needed to store the state of the
// absence period entitlement calculation prompt
```
did I understand your comment correctly? if yes, would you mind enhancing it a bit like suggested?